src/loaders/BufferGeometryLoader.js

```diff
@@ -56,8 +56,8 @@ THREE.BufferGeometryLoader.prototype = {
 
 			var attribute = attributes[ key ];
 			var typedArray = new TYPED_ARRAYS[ attribute.type ]( attribute.array );
-
-			geometry.addAttribute( key, new THREE.BufferAttribute( typedArray, attribute.itemSize ) );
+			var normalized = attribute.normalized || false;
```

Comment:
Unsigned integers were automatically normalized before, right?

Wondering whether `BufferAttribute` constructor should better check for `undefined` and in this case make sure user gets the old behavior...